tools/public_api_guard/material/tabs.d.ts

```diff
@@ -108,7 +112,8 @@ export declare class MatTab extends _MatTabMixinBase implements OnInit, CanDisab
     position: number | null;
     templateLabel: MatTabLabel;
     textLabel: string;
-    constructor(_viewContainerRef: ViewContainerRef);
+    constructor(_viewContainerRef: ViewContainerRef,
+    _closestTabGroup?: any);
```

Comment:
This is the way that clang wants to format this? any benefit to having this be two lines instead of one?

Reply:
It's not clang, it's what the API goldens script generated.